cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,241 @@
+   menu and under *Intelligence + analytics*, or directly through the *HDInsight clusters*
+   icon in the left sidebar):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-1.png
+      :figwidth: 100%
+      :width: 611px
+      :class: bordered-image
+
+      **Microsoft Azure:** Finding HDInsight
+
+#. Create an *HDInsight cluster* by clicking the "Create" button. *Quick create* will create a
+   cluster, but without CDAP installed. We recommend instead using the *Custom (size,
+   settings, apps)* tab to access additional options and add installing CDAP as part of
+   the cluster creation.
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.0.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster using *Quick create*
+
+   Using the *Custom (size, settings, apps)* tab, enter a *Cluster name* (such as
+   *cdap-cluster*). Under *Cluster type*, select "HBase". (To continue, click *Select* to
+   close the *Cluster configuration* window):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.1.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster using *Custom (size, settings, apps)* settings
+
+#. Enter the required basic settings: *Cluster login password*, *Resource group*, and *Location*.
+   As a minimum, a cluster requires 4 D3 v2 region nodes. This is enough YARN capacity to
+   run only simple examples. For any serious work, you will need a cluster with larger
+   nodes:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.0.png
+      :figwidth: 100%
+      :width: 674px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Configuring basic settings
+
+#. Under *Storage*, set a storage account, either an existing account or enter a unique
+   name for a new account.
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.1.png
+      :figwidth: 100%
+      :width: 676px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Setting a storage account
+
+#. Under *Applications*, search for *CDAP* and select the version you'd like to install
+   and accept the legal terms:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.2.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Setting CDAP as an application on the cluster
+
+#. Under *Cluster size*, at a minimum, a cluster requires four *D3 v2* region nodes. This is
+   enough YARN capacity to run only simple examples. For any serious work, you will need a
+   cluster with larger nodes:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.3.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Setting cluster size
+
+#. Under *Advanced settings*, there are currently no settings to be made:
```

Comment:
no settings required... (someone could still use them to specify their network or similar)